lisa/testsuite.py

```diff
@@ -539,10 +540,18 @@ def __init__(
         self.__log = get_logger("suite", metadata.name)
 
     def before_suite(self, log: Logger, **kwargs: Any) -> None:
-        ...
+        warnings.warn(
+            "before_suite is deprecated, use before_case instead",
+            DeprecationWarning,
+            stacklevel=2,
+        )
 
     def after_suite(self, log: Logger, **kwargs: Any) -> None:
-        ...
+        warnings.warn(
```

Comment:
deprecate is an annotation on methods, so it can raise error when writing code, instead of runtime.

Reply:
I didn't figure out a way that can give warning or error when user is writing code without running the code yet.

Reply:
I just found out the "deprecated" is not a standard lib. We can continue to use `warnings.warn`.